commands/certmanager/apiversion.go

```diff
@@ -0,0 +1,39 @@
+package certmanager
+
+import (
+	"context"
+
+	"github.com/ionos-cloud/ionosctl/pkg/core"
+	sdkgo "github.com/ionos-cloud/sdk-go-cert-manager"
+	"github.com/spf13/cobra"
+)
+
+func CertGetApiVersionCmd() *core.Command {
+	cmd := core.NewCommand(context.TODO(), nil, core.CommandBuilder{
+		Namespace:  "certmanager",
+		Resource:   "certificates",
+		Verb:       "api-version",
+		Aliases:    []string{"api", "info"},
+		ShortDesc:  "Get Certificate Manager API Version",
+		LongDesc:   "Use this command to retrieve API Version.",
+		Example:    "ionosctl certificate-manager api-version",
+		PreCmdRun:  core.NoPreRun,
+		CmdRun:     CmdGetApiVersion,
+		InitClient: true,
+	})
+
+	_ = cmd.Command.RegisterFlagCompletionFunc(ArgCols, func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		return allCols, cobra.ShellCompDirectiveNoFileComp
+	})
+
```

Comment:
add ArgNoHeaders, maybe I (an IonosCTL user) want to do a curl on the swagger / openapi-spec: I can't because I can't get rid of table headers